src/dialog/llm/abstract_llm.py

```diff
@@ -83,4 +82,6 @@ def process(self, input: str):
             "user_message": processed_input,
         })
         processed_output = self.postprocess(output)
+        if self.config.get("prompt").get("fallback_not_found_relevant_contents"):
+            return {"text": self.config.get("prompt").get("fallback_not_found_relevant_contents")}
```

Comment:
this if checks for zero documents retrieved? It seems like it will always fall here when "fallback_not_found_relevant_contents" is defined

Reply:
fixed!